gwdetchar/io/datafind.py

```diff
@@ -192,7 +193,7 @@ def get_data(channel, start, end, frametype=None, source=None,
         try:  # locate frame files
             ifo = re.search('[A-Z]1', frametype).group(0)
             obs = ifo[0]
-            source = gwdatafind.find_urls(obs, frametype, start, end)
+            source = io_datafind.find_urls(obs, frametype, start, end)
```

Comment:
Now that this is swapped, is `gwdatafind` used anywhere else? Should this be imported anymore?

Reply:
You're right that `gwdatafind` doesn't need to be imported anymore. I've gone ahead and removed the import statement